.travis.yml

```diff
@@ -1,36 +0,0 @@
-sudo: false
-
-cache:
-    directories:
-        - $HOME/.composer
-
-git:
-    depth: 1
-
-language: php
-
-before_install:
-    - pecl update-channels; pecl install grpc-1.15.0 > /dev/null
-    - composer self-update
-    - phpenv config-rm xdebug.ini || return 0
-    - composer global show hirak/prestissimo -q || composer global require hirak/prestissimo
-
-matrix:
-    include:
-        - name: PHP 7.2 with lowest versions of dependencies
-          php: '7.2'
-          install: composer update --prefer-lowest
-        - name: PHP 7.2
-          php: '7.2'
-          install: composer update
-        - name: PHP 7.3
-          php: '7.3'
-          install: composer update
-        - name: PHP 7.4
-          php: '7.4'
-          install: composer update
-
-script:
-  - vendor/bin/phpcs --standard=PSR2 --ignore=src/Grphp/grpc.stubs.php src
-  - vendor/bin/php-cs-fixer fix --diff --dry-run -v
```

Comment:
So we are losing code style checks? I think we have PSR-12 checks and should probably use it here too. What do you think?

Reply:
I added codesniffer and cs-fixer as CircleCI ckecks.